modules/howtos/examples/TransactionsExample.java

```diff
@@ -184,6 +193,18 @@ static void examples() {
                 TransactionGetResult docC = ctx.get(collection, "doc-c");
                 ctx.remove(docC);
 
+                // Performing a SELECT N1QL query against a scope:
+                QueryResult qr = ctx.query(inventory, "SELECT * FROM hotel WHERE country = $1",
+                        TransactionQueryOptions.queryOptions()
+                                .parameters(JsonArray.from("United Kingdom")));
+                List<JsonObject> rows = qr.rowsAs(JsonObject.class);
+
+                // Performing an UPDATE N1QL query against a scope:
+                ctx.query(inventory, "UPDATE hotel SET price = $1 WHERE name = $2",
```

Comment:
nitpick, but I would think it might be better to have something that updates multiple documents since that's a bit more realistic to where someone would likely use an UPDATE WHERE.  in fact, I might even mention this in a comment.

Reply:
Yes, fair point